service/service.go

```diff
@@ -173,7 +173,8 @@ func New(ctx context.Context, set Settings, cfg Config) (*Service, error) {
 		Logger:         logger,
 		MeterProvider:  mp,
 		TracerProvider: tracerProvider,
-		MetricsLevel:   cfg.Telemetry.Metrics.Level,
+		// Set MetricsLevel to Detailed during the transition period.
+		MetricsLevel: configtelemetry.LevelDetailed,
```

Comment:
Why? This may increase metrics for other unintentionally.

Reply:
It was an attempt to force people to use the new mechanism, but since it can be missed I reverted it